client/app/components/dialog-editor-modal-field-template/dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: until treepicker and API for it are in the SSUI -->
```

Comment:
@jntullo definitely not the second :)

As for `/api/automate_domains` .. yup, the list of domains seems to match the first level of that tree. But I don't see a way to expand methods, instances, or anything  under `/api/automate_domains/10000000000580/?expand=resources`, is there?